geoh5py/objects/object_base.py

```diff
@@ -309,7 +310,7 @@ def copy(
         if copy_children:
             children_map = {}
             for child in self.children:
-                if isinstance(child, PropertyGroup):
+                if isinstance(child, PropertyGroup | GeometricDataConstants):
```

Comment:
Can you explain what's that's `GeometricDataConstants` for my understanding? I understand it's data, but does it behave like PropertyGroup?

Reply:
It is specific to these
![image](https://github.com/user-attachments/assets/dc8fbbe5-6dcf-44a9-a858-c1ab60877d85)

Either the locations (created by GA) or the functions. Their respective DataType defines the behaviour.

Reply:
ok I got it